exporters/otlp/CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ if(WITH_OTLP_HTTP)
     opentelemetry_exporter_otlp_http_client
     PUBLIC opentelemetry_sdk opentelemetry_proto http_client_curl
            nlohmann_json::nlohmann_json)
+  if(nlohmann_json_clone)
+    add_dependencies(opentelemetry_exporter_otlp_http_client
+                     nlohmann_json::nlohmann_json)
+    list(APPEND OPENTELEMETRY_OTLP_TARGETS nlohmann_json_)
```

Comment:
Why is this step required?

Reply:
it's needed for install. `nlohmann_json::nlohmann_json` is an alias for `nlohmann_json_` here.

Reply:
ok, thanks. Should that be an install target for Zipkin and ETW too, or else just have it in `cmake/nlohmann-json.cmake` ?

Reply:
good idea. done